components/sections/Navbar.tsx

```diff
@@ -15,6 +15,11 @@ const links = [
     link: COMMUNITY_URL,
     newPage: true,
   },
+  {
```

Comment:
This page is highlighting the navbar issue. That will be needed to be pushed separately before going live.

![image](https://user-images.githubusercontent.com/7558499/168595806-54dc7fc0-d880-4c7e-85e7-2075fe6e8565.png)

Reply:
Hero component bg color changed to black